queries/ada/highlights.scm

```diff
@@ -0,0 +1,174 @@
+;; highlight queries.
+;; See the syntax at https://tree-sitter.github.io/tree-sitter/using-parsers#pattern-matching-with-queries
+;; See also https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#parser-configurations
+;; for a list of recommended @ tags, though not all of them have matching
+;; highlights in neovim.
+
+[
+   "abort"
+   "abs"
+   "accept"
+   "all"
+   "at"
+   "begin"
+   "declare"
+   "delay"
+   "until"
+   "do"
+   "end"
+   "entry"
+   "exit"
+   "generic"
+   "is"
+   "null"
+   "others"
+   "out"
+   "pragma"
+   "renames"
+   "task"
+   "terminate"
+   "when"
+] @keyword
+[
+   "abstract"
+   "access"
+   "aliased"
+   "array"
+   "constant"
+   "delta"
+   "digits"
+   "interface"
+   "limited"
+   "of"
+   "private"
+   "range"
+   "synchronized"
+   "tagged"
+] @StorageClass
+[
+   "mod"
+   "new"
+   "protected"
+   "record"
+   "subtype"
+   "type"
+] @type.definition
```

Comment:
I don't think `@type.definition` applies to any of those keywords. `@type.definition` is not a good name. It is thought for the identifier of a typdeffed definition